Evostory/EvoStory.BackendAPI/Repository/StoryRepositoryInMemory.cs

```
@@ -0,0 +1,49 @@
+﻿using Evostory.Story.Models;
+
+namespace EvoStory.BackendAPI.Repository
+{
+    public class StoryRepositoryInMemory : IStoryRepository
+    {
+        private List<Story> stories = new();
+        public void CreateStory(Story story)
+        {
+            stories.Add(story);
+        }
+
+        public void DeleteStory(Guid storyId)
+        {
+            var result = stories.FirstOrDefault(story => story.Id == storyId);
+            if (result == null)
+            {
+                throw new KeyNotFoundException($"No story with {storyId} found.");
+            }
+            stories.Remove(result);
+        }
+
+        public Story? GetStory(Guid storyId)
```

I find the error handling in this method strange. The method signature suggests that the return value can be null, but if it tries to be null, then an exception is thrown. What you need to decide is whether you want to throw an exception if the story is not found, or you wan to return null. Either one makes sense, but it's very confusing if you mix it.

Best idea here is to conform to how the rest of the guys are already doing. In our case that means you should agree with the other backend guys and you should all do it the same way.